ods_ci/test-variables.yml.example

```diff
@@ -77,6 +77,7 @@ COMPONENTS:
   codeflare: Removed
   ray: Removed
 PRODUCT: RHODS
+OPERATOR_NAME: rhods-operator
```

Comment:
Yeah, this is good, but we need to update our automation too to reflect this. Will share in PM.

Reply:
Could you please help me to identify which automation file you are refering here. I would verify these changes.

Reply:
sure